src/modules/drive/Toolbar/components/ShortcutCreationModal.spec.jsx

```diff
@@ -156,9 +170,11 @@ describe('ShortcutCreationModal', () => {
     fireEvent.change(filenameInput, { target: { value: '..' } })
     fireEvent.click(submitButton)
 
-    await waitFor(() => expect(Alerter.error).toHaveBeenCalledTimes(1))
-    expect(Alerter.error).toHaveBeenCalledWith('alert.file_name_illegal_name', {
-      fileName: '..'
+    await waitFor(() => {
+      expect(showAlert).toHaveBeenCalledTimes(1)
+      expect(showAlert).toBeCalledWith(
```

Comment:
I find that removing the verification on the message reduces test coverage. Was there a particular constraint for not migrating them?

Reply:
On pourrait simplement tester `message` mais on ne teste plus les valeurs des traductions.

Donc là pour faire un peu ce qui était fait avant et tester la clé de trad et les param passés, il faudrait mocker `t`. Ce qui n'est pas évident pour deux raisons : il y a pas mal d'export depuis `cozy-ui/transpiled/react/providers/I18n` et on utilise pas mal de ces exports dans Drive dans des sous-composants de celui testé.

Tout ça pour au final, à mon sens, pour quelque chose qui n'a pas une grande valeur ajoutée. Ce qui compte surtout c'est de savoir qu'il y a un message d'erreur qui apparaît.